exporter/awsemfexporter/pusher.go

```diff
@@ -342,3 +358,12 @@ func (p *pusher) renewLogEventBatch() *LogEventBatch {
 
 	return prevBatch
 }
+
+func (p *pusher) renewStructureLogClient() error {
+	awsConfig, session, err := awsutil.GetAWSConfigSession(p.logger, &awsutil.Conn{}, &p.config.(*Config).AWSSessionSettings)
+	if err != nil {
+		return err
+	}
+	p.svcStructuredLog = NewCloudWatchLogsClient(p.logger, awsConfig, p.appInfo, session)
```

Comment:
Think you need a lock when reading/writing now?

Reply:
yes. it is needed.